### ISSUE TITLE
Does this work with learning rate scheduler?

### ISSUE THREAD

**Issue by kota8** [surmenok/keras_lr_finder#7]
Thanks for the work!

I am wondering if this functionality works with learning rate scheduler (https://keras.io/callbacks/#learningratescheduler), and also custom scheduler like https://gist.github.com/jeremyjordan/5a222e04bb78c242f5763ad40626c452.

If not, I think this will be a desired extension.
